docs/components/advocacy_plan.md

```diff
@@ -0,0 +1,24 @@
+# Advocacy Plan
+
+## Summary
+
+Setting out steps to encourage organizations to adopt the standard.
+
+## Description
+
+An advocacy plan provides the resources and sets out the steps that will be followed to encourage organisations to adopt a standard, as well as key arguments. It should be updated regularly as the standard matures and as the standard starts to have an impact.
+
+## Prioritisation Factors
+
+- Where standards are being developed outside of the organisations that are expected to adopt the standard
+- If there is a perceived cost to adopting a standard
+
+## Deprioritisation Factors
+
+- Where a standard is emerging from consensus among the organisations that are expected to use it
+
+## Related Components
+
+[Blog](#blog)
```

Comment:
The Blog section was removed from the original page (components/index.md)  and now Communication channels describes blogs. 
What does this link link to? Is it for a future section?

Reply:
I can see @kathryn-ods 's on it :)